services/webrtc.ts

```
@@ -8,17 +8,31 @@
   private remoteStreams: Map<string, MediaStream> = new Map();
   private dataChannel: RTCDataChannel | null = null;
   private peerConfig: PeerConfig;
   private onIceCandidateCallback: ((candidate: RTCIceCandidate, connectionId: string) => void) | null = null;
-  private onNegotiationNeededCallback: (() => void) | null = null;
+  private onNegotiationNeededCallback: ((connectionId: string) => void) | null = null;
   private onTrackCallback: ((stream: MediaStream, peerId: string) => void) | null = null;
   private onDataChannelCallback: ((channel: RTCDataChannel) => void) | null = null;
+  private onTrackRemovedCallback: ((trackId: string, peerId: string) => void) | null = null;
   private currentConnectionId: string | null = null;
+  private localSenders: Map<string, TrackSender> = new Map();
+  private isRenegotiating: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)